src/TaskHandler.php

```diff
@@ -26,19 +26,22 @@ public function __construct(CloudTasksClient $client, Request $request, OpenIdVe
      */
     public function handle($task = null)
     {
-        $this->authorizeRequest();
-
         $task = $task ?: $this->captureTask();
 
-        $this->listenForEvents();
+        $command = unserialize($task['data']['command']);
+        $connection = $command->connection ?? 'cloudtasks';
```

Comment:
I had to flip the order here a little.  The payload is being unserialized and read for the connection value as it cannot be authorized without knowing which connection to use! :smile: